spacy/pipeline/entityruler.py

```diff
@@ -183,6 +185,27 @@ def labels(self) -> Tuple[str, ...]:
                 all_labels.add(l)
         return tuple(all_labels)
 
+    def initialize(
+        self,
+        get_examples: Callable[[], Iterable[Example]],
+        *,
+        nlp: Optional[Language] = None,
+        patterns_path: Optional[Path] = None
```

Comment:
We should make this `patterns` and have it take the actual patterns. Reading from the path should happen in the config, e.g. via `srsly.read_json.v1` (just like we currently do it for the attribute ruler etc.)

Reply:
Right, sorry, I lost track a bit of the latest refactors...

Reply:
Ok fixed (I hope), will also update original post to avoid later confusion